src/lando/ui/legacy/forms.py

```diff
@@ -12,7 +18,82 @@ class TransplantRequestForm(forms.Form):
     flags = forms.JSONField(widget=forms.widgets.HiddenInput, required=False)
 
 
-class UpliftRequestForm(forms.Form):
+class UpliftAssessmentForm(forms.Form):
```

Comment:
Reading https://docs.djangoproject.com/en/5.2/topics/forms/modelforms/, it might definitely be worth trying the `ModelForm`, this might cut out some boilerplate, particularly for those `Uplift*Forms`, which are similar but ever so slightly different.

One potential problem that I haven't found an option for is to display read-only fields, though https://docs.djangoproject.com/en/5.2/ref/forms/fields/#disabled may work, by specifically re-declaring those fields in the (Model)Form.

Reply:
I migrated the form to a `ModelForm` and it seems to Just Work. :tada: It also removed the need for a few helper methods and their associated tests.